packages/widgets/user-profile-widget/src/lib/widget/state/selectors.ts

```diff
@@ -17,3 +18,33 @@ export const getIsPhoneVerified = createSelector(
 );
 export const getHasPasskey = createSelector(getMe, (me) => me.webauthn);
 export const getHasPassword = createSelector(getMe, (me) => me.password);
+
+export const getCustomAttributes = (state: State) =>
+  state.customAttributes.data;
+
+export const getUserCustomAttrs = createSelector(
+  getMe,
+  getCustomAttributes,
+  (userData, allCustomAttrs = []) => {
+    const res: Record<string, string> = {};
+    const userCustomAttributes = userData['customAttributes'] || {};
+
+    Object.keys(userCustomAttributes).forEach((key: string) => {
```

Comment:
I think that in this case, it will be better to use reduce on Object.entries

```
Object.entries(userCustomAttributes).reduce((acc, [key, val]) => { ...
```